ops/ansible/roles/bfd-db-migrator/templates/migrator-monitor.sh.j2

```diff
@@ -65,18 +65,46 @@ function get_migrator_state {
 #######################################
 function migrator_is_running {
   local _state="$1"
-  local _code="$(jq -r .code <<< "$_state")"
-  local _status="$(jq -r .status <<< "$_state")"
+
+  local _code
+  local _status
+  _code="$(jq -r .code <<< "$_state")"
+  _status="$(jq -r .status <<< "$_state")"
   [ "$_code" = '(null)' ] && [ "$_status" = "0/0" ]
 }
 
+#######################################
+# Sets the migration schema version based on the monitor log
+# Globals:
+#   None
+# Arguments:
+#   Current state of the migrator
+# Outputs:
+#  json response to be used to update the schema version
+#  that was reported out in the monitor log and report that schema version
+#  to be used to send to SQS in order to update the migrator on whether to deploy or not
+#######################################
+function set_migrated_state {
+  local _state="$1"
+  local schema_version
+
+  if grep -q "No migration necessary." "{{ db_migrator_dir }}/migrator-log.json"; then
```

Comment:
Nitpick, but can we make `"{{ db_migrator_dir }}/migrator-log.json"` a variable?

Reply:
> Nitpick, but can we make `"{{ db_migrator_dir }}/migrator-log.json"` a variable?

I think this is _somewhat_ out of scope, but better is better. What did you have in mind? 

Were you thinking a _global_ shell variable, e.g. 

```sh
MIGRATOR_LOG="{{ db_migrator_dir }}/migrator-log.json"
# ..snipped..
  if grep -q "No migration necessary." "$MIGRATOR_LOG"; then
# ..snipped..
```

Maybe as an ansible variable, used in e.g.

```sh
if grep -q "No migration necessary." "{{ db_migrator_log_file }}"; then
```

Some combination of the above, or... something else entirely?

Reply:
^^ please